Lib/Gps/gps.cpp

```diff
@@ -0,0 +1,91 @@
+//
+// Created by Filip on 20.04.2021.
+//
+
+#include "Inc/Gps/gps.hpp"
+
+#include <terminal.hpp>
+#include <vector>
+
+namespace habsat::gps{
+
+Gps::Gps(UART_HandleTypeDef *uart, DMA_HandleTypeDef* dma_usart_rx) : uart(uart), dma_rx(dma_usart_rx){
+    init();
```

Comment:
don't do init in constructors. we will need to be able to control the initialization process later on